src/components/Invoice/params.schema.ts

```diff
@@ -0,0 +1,5 @@
+import { z } from 'zod';
+
+export const invoiceParamsSchema = z.object({
```

Comment:
It could be also placed inside the component file, as it will only be used there I think

Reply:
done